### ISSUE TITLE
[Feature request] Allow posting to forum channels on Discord

### ISSUE THREAD

**Issue by LeviSnoot** [mvdicarlo/postybirb#151]
As of right now on 3.1.6, PB errors out when attempting to post to a forum channel on my server. Here's the log;

```json
   "response": {
    "website": "Discord",
    "time": "12/22/2022, 4:55:30 PM",
    "message": "Webhook Failure",
    "error": null,
    "additionalInfo": {
     "message": "Webhooks posted to forum channels must have a thread_name or thread_id",
     "code": 220001
    }
```

So it looks like PostyBirb just needs to make the post title the `thread_name` for forum posts. It would also be good to implement forum tags as it is required in my forum channel to use at least one tag to post there. Thanks for the awesome work on PB!

**Comment by kiit0s** [comment]
just encountered this issue myself today, I'd love to see this functionality added!

**Comment by LeviSnoot** [comment]
> just encountered this issue myself today, I'd love to see this functionality added!

So apparently there's a [postybirb-plus](https://github.com/mvdicarlo/postybirb-plus) repo which is where development seems to be focused. Not sure how I missed that when making this thread initially. That said it looks like this issue has been fixed in [#199](https://github.com/mvdicarlo/postybirb-plus/pull/199) on that repo, so I will close this issue since PB+ is what I use anyway.
